xapian-applications/omega/handler_poppler.cc

```diff
@@ -18,12 +18,11 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
  * USA
  */
-
 #include <config.h>
+
```

Comment:
It's better to take care to avoid this sort of arbitrary whitespace change in patches (it can easily result from amending a patch, like here, or from not taking care with what's included in a patch in the first place).  It's a good habit to review the patch against the branch point before pushing (both initially and before pushing fixup commits).

It's distracting noise for the patch reviewers, but also makes it harder to look back through the history of the code - e.g. `git blame` would show this code as having changed.

Reply:
It appears with the remove/add of config.h. I didn't notice.
I will try to be more careful with this kind of thinks.